miniteltest/expectations.go

```diff
@@ -179,6 +180,16 @@ func (ts *TestServer) Wait(max time.Duration) bool {
 	}
 }
 
+// ExpectDone returns a non-nil error when unfulfulled expectations are present in
+// the test server.
+func (ts *TestServer) ExpectDone() error {
```

Comment:
FWIW: I prefer to have the test fail locally instead of in this function. If we want to pass a t, do it to ExpectDone and have it not return an error.